Icon sizes changed after upgrading

**Describe the bug**
After upgrading to `0.3.0` I noticed that the size of some icons changed. For instance, `ellipsis-v` is displayed much smaller than expected, and `angle-down`/`angle-up` also look smaller.

**Reproducible test case**
Upgrade to 0.3.0 then use `<FaIcon @icon="sliders-h" />`.

**Expected behavior**
I ran a diff between the icons rendered in `0.2.3` and `0.3.0`:
<img width="1085" alt="Screen Shot 2022-01-24 at 5 14 35 PM" src="https://user-images.githubusercontent.com/795927/150821143-35c28f0a-89ef-4b28-891b-703372ce9ddb.png">
The lack of ID and ember-view class is expected with Octane, but the viewBox should be identical.

**Desktop**
 - Browser Chrome
 - Version 97

I hope there was enough helpful information? What's weird is that it's only happening with a few icons 🤔

There you go. Looks good to me when checking the dummy app. 

Unfortunately I was unable to `yarn link` the addon because it resulted I weird build error. So I could not test if everything is fixed within my app.

```
Build Error (broccoli-fontawesome-pack: pro-solid-svg-icons)

Cannot find module '@fortawesome/pro-solid-svg-icons'
Require stack:
- ember-fontawesome/vendor/broccoli-fontawesome-pack.js
- ember-fontawesome/index.js
```

I also was unable to test the icon where I was noticing the issue because it is not included in the free font awesome package and I didn't want to go through all the mumbo jumbo to make the paid icons work with the dummy app. 

And lastly (off topic): I do hate the icons.js file. What's even worse is that editing it takes a whole rebuilt of the app during development.

Now, can we get a release please.

--- a/addon/components/fa-icon.js
+++ b/addon/components/fa-icon.js
@@ -134,6 +134,6 @@ export default class FaIconComponent extends Component {
   }
 
   get viewBox() {
-    return this.abstractIcon?.viewBox ?? '0 0 448 512';
+    return this.abstractIcon?.attributes?.viewBox ?? '0 0 448 512';
   }
 }

